config/schema/og.schema.yml

```diff
@@ -1,5 +1,5 @@
 og.settings:
-  type: config_object
+  type: config_entity
```

Comment:
This just moves the problem to somewhere else. This config type is not the issue.